frontend/apps/crates/entry/jig/play/src/player/dom.rs

```diff
@@ -165,6 +180,22 @@ pub fn render(state: Rc<State>) -> Dom {
     })
 }
 
+/// Emits `true` if the module doesn't exist in the list of modules or if the jig is `None`.
+/// Otherwise emits the value of the module's `is_complete` field.
+fn active_module_valid_signal(state: Rc<State>) -> impl Signal<Item = bool> {
+    jig_and_active_module_signal(state).map(|(jig, active_module_index)| {
+        match jig {
+            Some(jig) => match &jig.jig_data.modules.get(active_module_index) {
+                Some(module) => {
+                    module.is_complete || matches!(module.kind, ModuleKind::Legacy)
+                },
+                None => true, // Active module isn't in the list
+            },
+            None => true, // Jig isn't set
+        }
+    })
```

Comment:
To prevent breakage with legacy modules and debug, jigs where are None and modules which are Legacy are valid.